staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config_test.go

```diff
@@ -2170,10 +2176,14 @@ func validState(t *testing.T, keyID string, exp time.Time, useSeed bool) envelop
 		t.Fatal(err)
 	}
 	return envelopekmsv2.State{
-		Transformer:         transformer,
-		EncryptedObject:     *encObject,
-		ExpirationTimestamp: exp,
-		CacheKey:            cacheKey,
+		Transformer:                           transformer,
+		EncryptedObjectEncryptedData:          encObject.EncryptedData,
```

Comment:
drop the EncryptedObjectEncryptedData field, we never set anything in it in `state` and always populated it later:

```
	o.EncryptedData = nil // make sure that later code that uses this encrypted object sets this field
```